packages/material-ui/src/Alert/Alert.js

```diff
@@ -180,29 +254,24 @@ const Alert = React.forwardRef(function Alert(props, ref) {
   );
 
   return (
-    <Paper
+    <AlertRoot
       role={role}
       square
       elevation={0}
-      className={clsx(
-        classes.root,
-        classes[variant],
-        classes[`${variant}${capitalize(color || severity)}`],
-        themeVariantsClasses,
-        className,
-      )}
+      styleProps={styleProps}
+      className={clsx(classes.root, classes[variant], themeVariantsClasses, className)}
```

Comment:
```suggestion
      className={clsx(classes.root, className)}
```

The variant classes should be included already in the `classes.root`